src/ansible/roles/keycloak/tasks/main.yml

```diff
@@ -0,0 +1,119 @@
+- name: Get Keycloak tags
+  uri:
+    url: https://api.github.com/repos/keycloak/keycloak/tags
```

Comment:
Thinking of this, it may be safer to rely on latest release than on tags, ie `https://api.github.com/repos/keycloak/keycloak/releases/latest` and read the tag name from there - or read tarball_url directly if you don't require specific asset but can work with the full source code.

Reply:
I switched this to use latest but the Fedora 39 pipeline failed on this.   I'm rerunning now.

Reply:
ok, f39 worked so marking this resolved.